t/ui/03-source.t

```diff
@@ -23,18 +23,24 @@ use Test::Mojo;
 use Test::Warnings;
 use OpenQA::Test::Case;
 
-SKIP: {
-    skip "breaks package build", 4;
-
-    OpenQA::Test::Case->new->init_data;
-
-    my $t = Test::Mojo->new('OpenQA::WebAPI');
-
-    my $test_name = 'isosize';
-
-    $t->get_ok("/tests/99938/modules/$test_name/steps/1/src")->status_is(200)
-      ->content_like(qr|inst\.d/.*$test_name.pm|i, "$test_name test source found")
-      ->content_like(qr/ISO_MAXSIZE/i,             "$test_name test source shown");
-}
-
-done_testing();
+my $schema  = OpenQA::Test::Case->new->init_data;
+my $t       = Test::Mojo->new('OpenQA::WebAPI');
+my $name    = 'installer_timezone';
+my $id      = 99938;
+my $src_url = "/tests/$id/modules/$name/steps/1/src";
+$t->get_ok($src_url)->status_is(200)
+  ->content_like(qr|installation/.*$name.pm|i, "$name test source found")
+  ->content_like(qr/assert_screen.*timezone/i, "$name test source shown");
+
+subtest 'source view for jobs using VCS based tests' => sub {
+    # simulate the job had been triggered with a VCS checkout setting
+    my $jobs        = $schema->resultset('Jobs');
+    my $settings_rs = $jobs->find($id)->settings_rs;
+    my $casedir     = 'https://github.com/me/repo#my/branch';
+    $settings_rs->update_or_create({job_id => $id, key => 'CASEDIR', value => $casedir});
+    $t->get_ok($src_url)->status_is(302);
+    my $expected = qr@github.com/me/repo/blob/my/branch/tests.*/installer_timezone@;
+    $t->tx->res->headers->location($expected, 'remote module is referenced');
```

Comment:
There's also a `header_like` method.
```perl
$t->get_ok(...)->status_is(302)->header_like('Location' => $expected);
```

Reply:
yes, I should use that. What I am doing in this line looks completely broken :D